src/helpers/ParseQuery.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Returns what the input is (zip code, county name, or address)
+ * @Param str The string to parse
+ * @Returns A string telling what the input is
+ */
+export default function Parse(str: string): string {
+  const zip = new RegExp('^\\d{5}$');
+  // let county = new RegExp('(.+), ([a-z]{2})');
+  // let county: RegExp = /(.+), ([a-z]{2})/;
+  const county = new RegExp('(^[A-z]+), ([A-z]{2})');
+  const add1 = new RegExp('(.+), ([A-z]{2})');
```

Comment:
It took me a while before I got what add1 stands for (later in the code). I think that it would be easier for us in the future to make these tags self-explanatory (if that is a word). Maybe 'address' for now?
```suggestion
  const address = new RegExp('(.+), ([A-z]{2})');
```

Reply:
Sure! I left it add1 because I was thinking we might add a couple of more regex for more coverage later on at some point.